uint8 imu_type
int16 temp
uint8 imu_conf